ui-super-heroes/README.md

```diff
@@ -23,34 +23,39 @@ Environment variables can be injected into the build using the [ngx-env](https:/
 
 Variables must start with the `NG_APP` prefix, e.g `NG_APP_MY_URL=http://localhost:1234`.
 
-Production builds are served using a Node.js server. This server serves the compiled Angular application and an `env.js` file. This `env.js` file is generated at startup, and adds a `window.NG_CONFIG` property that the Angular application can read from.
+Production builds are served using a Quarkus server. This server serves the compiled Angular application and an `env.js` file. This `env.js` file is generated at startup, and adds a `window.NG_CONFIG` property that the Angular application can read from.
 
 Currently, the `env.js` will expose just the `API_BASE_URL` that's set at runtime. This will control the base URL to connect to the [fights](../rest-fights) service. The default if unset is http://localhost:8082.
 
-You also need to make sure the angular CLI is installed (`npm install @angular/cli@12.2.8`).
 
 ```bash
-npm run build
+quarkus package
+```
+
+It is also possible to build a native binary, using 
+
+```bash
+./mvnw -B clean package -DskipTests -Pnative
 ```
 
 ## Local Development
 Use the following command:
 
 ```shell
-npm run dev
+quarkus dev
 ```
 
-This starts the Angular hot reloading server at http://localhost:4200, and the Node.js server to supply the `env.js` file. The Angular server on port 4200 will proxy the request for `env.js` to the Node.js server on port 8080.
+This starts both Quarkus and the Angular hot reloading server at http://localhost:4200. The Quarkus server to supplies the `env.js` file to the Javascript front-end. 
 
-The Node.js server port can be changed by setting the `HTTP_PORT` variable. The `ng.proxy.config.json` file will need to be updated with the new Node.js server port number if you deviate from 8080.
+The Quarkus server port can be changed in the usual way, with `application.properties`. 
 
 ## Running the Application
 1. First you need to start up all of the downstream services ([Heroes Service](../rest-heroes), [Villains Service](../rest-villains), and [Fights Service](../rest-fights)). 
     - The [Event Statistics Service](../event-statistics) is optional.
 2. Follow the steps above section, *Building the Application*.
 3. Set the `API_BASE_URL` environment variable with the appropriate [Fights Service](../rest-fights) hostname and port.
    > By default, the [`rest-fights`](../rest-fights) service runs on port `8082`, so setting `API_BASE_URL=http://localhost:8082` will do.
-4. Start the service using the command `npm start`.
+4. Start the service using the command `quarkus dev`.
```

Comment:
I think that's a good idea, but maybe for another work item? I was trying to keep this one as a like-for-like swap, so that the new service behaved the same as the old one.

Reply:
I thought the other app did default to localhost:8082? Or maybe it didn't? Honestly I don't remember :)

Reply:
Ah, I see what you mean. You're right, it should run correctly with no config. The readme even says so: 

```
Currently, the `env.js` will expose just the `API_BASE_URL` that's set at runtime. This will control the base URL to connect to the [fights](../rest-fights) service. The default if unset is http://localhost:8082.
```

That's already implemented, with this, in `EnvResource.java`

```
    @ConfigProperty(name = "api.base.url", defaultValue = "http://localhost:8082")
    String url;
 ```   
 
 The readme should probably be updated to explain that it works by normal Quarkus config so you can use `api.base.url` or an env variable of `API_BASE_URL`.